OculusFireConfig fireMessage
uint32  pingId
uint32  status
float64 frequency
float64 temperature
float64 pressure
float64 speeedOfSoundUsed
uint32  pingStartTime
uint8   dataSize
float64 rangeResolution
uint16  nRanges
uint16  nBeams
uint32  imageOffset
uint32  imageSize
uint32  messageSize

uint8[] data

